packages/0xcert-wanchain-utils/src/lib/normalize-address.ts

```diff
@@ -0,0 +1,30 @@
+import { normalizeAddress as normalizeEthereumAddress } from '@0xcert/ethereum-utils/dist/lib/normalize-address';
+
+/**
+ * Converts Wanchain address to checksum format.
+ * NOTE: Wanchain uses basically the same mechanism, you only have to inverse
+ * lower and upper case on characters.
+ */
+export function normalizeAddress(address: string): string {
+  if (!address) {
+    return null;
+  }
+
+  // We are using ethers.js for encoding ABI calls. This means that if we send in a wanchain address,
```

Comment:
`We are using ethers.js for encoding ABI calls. This means that if we send using a Wanchain address,`